code/game/objects/items/devices/lightreplacer.dm

```diff
@@ -66,6 +65,15 @@
 	// when we get this many shards, we get a free bulb.
 	var/shards_required = 4
 
+
+//redds engiborg light replace because fuck you
+/obj/item/lightreplacer/cyborg/engineer
```

Comment:
```suggestion
/obj/item/lightreplacer/cyborg/engineer //redds engiborg light replacer
```

Reply:
Still kind of weird to even comment on this because comments are meant to clarify the code.
Adding a "this is my personal line of code" doesn't add anything to our codebase in terms of value.

Either remove this or if you're really hell bent on keeping this commit it on to 1 line